PyInstaller/utils/hooks/__init__.py

```diff
@@ -18,12 +18,14 @@
 from pathlib import Path
 from typing import Tuple
 
-from ...compat import base_prefix, exec_command_stdout, exec_python, \
+from PyInstaller.compat import base_prefix, exec_command_stdout, exec_python, \
```

Comment:
Any reason why this one was not turned into from `PyInstaller import compat` like elsewhere?

Reply:
Because that namespace is very public. I expect at least one person somewhere will have mistakenly thought one of those `compat` functions are hook utilities and imported them from `PyInstaller.utils.hooks`.

Reply:
Actually, looking at those names, none of them look particularly useful for hooks anyway so I guess the chances of them being used accidentally are slim. `is_darwin` would probably be quite handy if the other `is_[platform]`s were there too but they're not.

Reply:
Yeah sod it. I'll change it as you said.